.github/workflows/main.yml

```diff
@@ -2,6 +2,22 @@ name: tests
 
 on: [push, pull_request]
 
+env:
+  FORCE_COLOR: 1  # Request colored output from CLI tools supporting it
+  MYPY_FORCE_COLOR: 1  # MyPy's color enforcement
+  PIP_DISABLE_PIP_VERSION_CHECK: 1
+  PIP_NO_PYTHON_VERSION_WARNING: 1
+  PIP_NO_WARN_SCRIPT_LOCATION: 1
+  PY_COLORS: 1  # Recognized by the `py` package, dependency of `pytest`
+  TOX_PARALLEL_NO_SPINNER: 1
```

Comment:
Tox should probably support this by default. If it's known to be noisy in GHA, it should detect GHA and disable it automatically. Can you file a request please?

Reply:
IIRC it was rather controversial when it was first implemented in tox. Some toggles are only changeable via CLI or env vars, but not though config settings. The maintainers weren't open to improving this.